src/types/gqlObjects.ts

```diff
@@ -2783,7 +2753,7 @@ export const HmisObjectSchemas: GqlSchema[] = [
         type: {
           kind: 'NON_NULL',
           name: null,
-          ofType: { kind: 'SCALAR', name: 'ISO8601DateTime', ofType: null },
+          ofType: { kind: 'SCALAR', name: 'ISO8601Date', ofType: null },
```

Comment:
Hmm looks like these got accidentally reverted by a PR that theron merged recently. So we can keep them, these changes should go back into 106